.ci/checker-framework.groovy

```diff
@@ -27,14 +27,14 @@ System.exit(exitCode)
  * @param argument command line argument
  * @return {@code 0} if command executes successfully, {@code 1} otherwise
  */
-private int parseArgumentAndExecute(String argument) {
+private int parseArgumentAndExecute(final String argument) {
     final int exitCode
-    final Set<String> profiles = getCheckerFrameworkProfiles()
+    final Set<String> profiles = checkerFrameworkProfiles
```

Comment:
I am personally liking `()` for method calls. Makes it clear to differentiate methods and fields/variables.

Reply:
I always prefer to write in Groovy like in Java, to be ready to convert script to java project easily and do not deal with remembering syntax sugar of Groovy.

Reply:
Reverted it.